src/main/java/com/faforever/api/data/listeners/MapChangeListener.java

```diff
@@ -0,0 +1,20 @@
+package com.faforever.api.data.listeners;
+
+import com.faforever.api.data.domain.Map;
+import com.faforever.api.data.domain.MapVersion;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.PostRemove;
+import javax.persistence.PostUpdate;
+
+@Component
+public class MapChangeListener {
+
+  @CacheEvict(allEntries = true, cacheNames = {Map.TYPE_NAME, MapVersion.TYPE_NAME})
+  @PostUpdate
+  @PostRemove
+  public void mapChanged(Map map) {
+    //No action needed, evicts cache by annotation
```

Comment:
It's great when things are done automatically. It's awful if you have no chance to check when it happened.
So add a debug statement please.